server-setup/installing-and-connecting.md

```diff
@@ -81,11 +81,11 @@ Other IMUs, such as the BNO085 or ICM20948 do not require any specific manual ca
 
 ![img](../assets/img/Setup_NotDone.png)
 
-6. At this time, automatic mount calibration isn't ready, so click **Manually set mounting**. With your trackers on, click either **Left**, **Right**, **Front**, or **Back** depending on how you have your trackers mounted. There's no right or wrong answers, just whatever feels most comfortable for you! Click **Next step** once you've set the mounting orientation of all of your trackers. More information can be found on mounting on the [Putting on your trackers page](putting-on-trackers.html).
+6. With your trackers on, get into the reset pose below and click the **Reset Mounting** button, which will automatically calibrate the orientation of the trackers. Click **Next step** once you've set the mounting orientation of all of your trackers.
```

Comment:
The reset mounting part in the setup (not the small button top right) is in the works, so this'll need rewriting
(what it looks like: https://cdn.discordapp.com/attachments/823658497454374962/1047636992848052355/image.png)